instrumentation/opentelemetry-instrumentation-openai-v2/tests/conftest.py

```diff
@@ -58,7 +81,8 @@ def instrument():
 
 @pytest.fixture(scope="session", autouse=True)
 def uninstrument():
-    OpenAIInstrumentor().uninstrument()
+    # OpenAIInstrumentor().uninstrument()
```

Comment:
TODO: This was running before the test, will need to figure out how to do clean up right

Reply:
How does it fail?

Reply:
To me the correct way to handle this would be to do it per test:

```
@pytest.fixture(autouse=True)
def instrument():
    instrumentor = OpenAIInstrumentor()
    instrumentor.instrument()

    yield instrumentor

    instrumentor.uninstrument()
```

Reply:
thanks! will try!  how it failed - the instrumentation was disabled (since uninstrument was called before test started),